### DISCUSSION TITLE
Rust client

### DISCUSSION THREAD

**Discussion by matteobollettini** [rustdesk/rustdesk-server-pro#495] [Q&A]
Hi everyone

I want to replace Teamviewer with Rust Desk in my company for the remote support. With Teamviewer i made my customers download the quick support app for a one time remote support (without installation, only execute). When the remote support ended, the customer close the quick support app and nothing remains on my Teamviewer console (not a device remains "logged", but is a one time remote control).

Now i see that in the Rust Console the connections made remains as if they were fixed devices that i have to insert into my license. It is the normal operation of Rust (so i have to consider all the computers i may remote support to buy the right license)? Or is there a way to keep the devices from remaining in the list ad occupying a license?

I hope i explained myself correctly
Thansk for the support

**Comment by rustdesk** [comment]
> Or is there a way to keep the devices from remaining in the list ad occupying a license?

No yet

There are two solutions for you.

https://github.com/rustdesk/rustdesk-server-pro/discussions/182#discussioncomment-9082528
or 
https://github.com/rustdesk/rustdesk-server-pro/discussions/182#discussioncomment-10083053

**Reply by matteobollettini** [reply]
Thanks

I have to install Phyton on the same machine where i've installed Rust Server? Or another machine?
